ports/servoshell/desktop/app_state.rs

```diff
@@ -109,6 +109,9 @@ impl RunningAppState {
     pub(crate) fn new_toplevel_webview(self: &Rc<Self>, url: Url) {
         let webview = self.servo().new_webview(url);
         webview.set_delegate(self.clone());
+        if self.inner_mut().focused_webview_id.is_none() {
+            self.inner_mut().focused_webview_id = Some(webview.id());
+        }
```

Comment:
I looked into this a bit and I think it will be hard to fix this issue before we do more changes in the compositor (having a compositor per WebView and setting up the association synchronously when creating WebViews). In the meantime maybe we can look into just setting `focused_webview_id` as the first WebView we create when creating `AppState`.

Reply:
I've slightly modified this check.